content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,100 @@
-[among others](/doc/command-reference/remote/add#supported-storage-types)).
-
-> For hands-on experience, we recommend following the
-> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
-
-## DVC is not Git!
-
-DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories for versioning (among other purposes). They point to
-specific data contents in the <abbr>cache</abbr>, providing the ability to store
-multiple data versions out-of-the-box.
-
-Full-fledged
-[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
-
-Let's say you have an empty <abbr>DVC repository</abbr> and put a dataset of
-images in the `images/` directory. You can start tracking it with `dvc add`.
-This generates a `.dvc` file, which can be committed to Git in order to save the
-project's version:
-
-```dvc
-$ ls images/
-0001.jpg 0002.jpg 0003.jpg 0004.jpg ...
-
-$ dvc add images/
-
-$ git add images.dvc .gitignore
-$ git commit -m "Track images dataset with DVC."
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models? What’s the best way to organize and store
+multiple versions of data files for safe, persistent access? How can the
+intertwined lifecycles of data and models be followed and enforced?
+
+![](/img/data_ver_complex.png) _Exponential complexity of DS projects_
+
+DVC proposes to _codify_ data projects in order to adopt existing engineering
+tools like Git
+[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control),
+continuous integration (CI), and other best practices that improve productivity.
+This means writing a description of which data, ML artifacts, etc. should be in
+the environment at a given time. DVC can restore the <abbr>workspace</abbr>
+files and directories (from a separate data storage) to match that description.
+
+Data versioning lets us identify exact research inputs later, so anyone can
+understand and reproduce results. DVC brings additional benefits on top of that:
+make projects easier to work on by separating code from data (which also
+increases data persistence); enforce lifecycle and security policies by having
+all changes go though a Git repo (immutable history); collaborate with a toolset
+everyone is familiar with (data scientists, engineers, managers), and others
+(see [Get Started](/doc/start)).
```

Comment:
Should I turn this back into a bullet list? It's kind of boring just a paragraph after the next, and so on.